pypxe-server.py

```diff
@@ -85,9 +89,22 @@
             dargs.update(loadedcfg)
             args = argparse.Namespace(**dargs)
 
+        # setup logger
+        logger = logging.getLogger(sys.argv[0])
```

Comment:
This the logger name sys.argv[0] is the command name.. maybe we can use a fixed string like "PyPXE".